inspec.yml

```diff
@@ -0,0 +1,377 @@
+
+  - name: app_password_age # V-73231
+    desc: "Set the number of days that manually managed application account passwords must be changed"
+    type: Numeric
+    profile: microsoft-windows-server-2016-stig-baseline
+    <% if ['High-HVA', 'Moderate-HVA', 'Low-HVA'].include? ENV['BASELINE'] %>
+    value: 60
+    <% end %>
+ 
+  - name: maximum_password_age_machine # V-73641
+    desc: "Set the machine account maximum password age stated by your organiztion"
+    type: Numeric
+    profile: microsoft-windows-server-2016-stig-baseline
+    <% if ['High-HVA', 'Moderate-HVA', 'Low-HVA'].include? ENV['BASELINE'] %>
+    value: 60
+    <% end %>
+
+  - name: unused_account_age # V-73259
+    desc: "Set the maximum age in days for unused accounts stated by your organization"
+    type: Numeric
+    profile: microsoft-windows-server-2016-stig-baseline
+    <% if ['High-HVA', 'High'].include? ENV['BASELINE'] %>
+    value: 30
+    <% elsif ['Moderate-HVA', 'Moderate'].include? ENV['BASELINE'] %>
+    value: 60
+    <% elsif ENV['BASELINE'].nil? %>
+    value: 60
+    <% end %>
+
+  - name: minimum_password_length_manual # V-73229
+    desc: "Set the minimum password length for manually managed applications stated by your organization"
+    type: Numeric
+    profile: microsoft-windows-server-2016-stig-baseline
+    <% if ['High-HVA', 'Moderate-HVA', 'Low-HVA'].include? ENV['BASELINE'] %>
+    value: 20
+    <% elsif ['High', 'Moderate', 'Low'].include? ENV['BASELINE'] %>
+    value: 15 # Same as the baseline parameter
+    <% elsif ENV['BASELINE'].nil? %>
+    value: 15
+    <% end %>
+
+  - name: minimum_password_length # V-73321
+    desc: "Set the minimum password length stated by your organization"
+    type: Numeric
+    profile: microsoft-windows-server-2016-stig-baseline
+    <% if ['High-HVA', 'Moderate-HVA', 'Low-HVA'].include? ENV['BASELINE'] %>
+    value: 20
+    <% elsif ['High', 'Moderate', 'Low'].include? ENV['BASELINE'] %>
+    value: 15
+    <% elsif ENV['BASELINE'].nil? %>
+    value: 15
+    <% end %>  
+
+  # V-73319 minimum_password_age input value is the same as the baseline parameter of 1
+  # V-73317 maximum_password_age input value is the same as the baseline parameter of 60
+  
+  - name: password_history_size # V-73315
+    desc: "Set the password history size stated by your organization"
+    type: Numeric
+    profile: microsoft-windows-server-2016-stig-baseline
+    <% if ['High-HVA', 'Moderate-HVA', 'Low-HVA'].include? ENV['BASELINE'] %>
+    value: 24 # Max value to disallow password reuse
+    <% end %>
+
+  - name: maximum_idle_time #  V-73387
+    desc: "Set the maximum connection idle time stated by your organization"
+    type: Numeric
+    profile: microsoft-windows-server-2016-stig-baseline
+    <% if ['High-HVA', 'High', 'Moderate-HVA', 'Moderate'].include? ENV['BASELINE'] %>
+    value: 1800
+    <% elsif ENV['BASELINE'].nil? %>
+    value: 1800
+    <% end %>
+
+  - name: maximum_idle_time_phrase #  V-73387
```

Comment:
Can we calculate within the control so we don't need this input?

Reply:
Will need help with that.